pennylane/beta/tapes/qnode.py

```diff
@@ -454,8 +449,12 @@ def to_autograd(self):
 def qnode(device, interface="autograd", diff_method="best", **diff_options):
     """Decorator for creating QNodes.
 
-    When applied to a quantum function, this decorator converts it into
-    a :class:`QNode` instance.
+    This decorator is used to indicate to PennyLane that the decorated function contains a
+    :ref:`quantum variational circuit <glossary_variational_circuit>` that should be bound to a
+    compatible device.

```

Comment:
:100:

Reply:
lol, github was being glitchy, it's just 100, not 300 :smile:

Reply:
😆